content/en/blog/2024/demo-skyscanner/index.md

```diff
@@ -0,0 +1,96 @@
+(TTM), which goes from when a first responder acknowledges the incident to the
+time users stop suffering from the incident.
+
+## Environment
+
+To begin with, we need to set up an environment that demonstrates the best
+practices for monitoring and debugging using OpenTelemetry instrumentation and
+observability. For this, we propose the use of the official
+[OpenTelemetry Demo project](https://github.com/open-telemetry/opentelemetry-demo),
+which is a realistic example of a distributed system called Astronomy Shop.
+Thanks to its OpenTelemetry Protocol (OTLP), it allows us to simply point the
+standard OTLP exporter in the Collector to [New Relic](https://newrelic.com/),
+our chosen observability platform at Skyscanner which, like other platforms, is
+fully embracing open standards to ingest telemetry data.
+
+This system contains regressions that can be injected into the platform and
+helps us demonstrate the importance of Service Levels Objectives (SLOs),
+tracing, logs, metrics, etc. For instance, we can observe traffic flow through
+various components, as shown in the image below. Since part of the OpenTelemetry
+ecosystem is open-source, we can easily introduce any new features that will be
+reviewed by OpenTelemetry contributors.
+
+![Distributed tracing example in Astronomy shop](tracing_example.png)
+
+## Observability game day
+
+Once the environment is set up, we can introduce the Observability Game Day, an
+initiative based on the Wheel of Misfortune practices that Google uses and
+describes in the [Site Reliability Engineering book](https://sre.google/books/).
+
+This game simulates a production incident, where a moderator known as the game
+master (GM) conducts the session and someone from the audience spins the wheel
+and explains an incident or outage. The participants are then divided into teams
+and tasked with identifying and resolving the issue as quickly as possible. If
+the solution is not optimal, the GM can help by introducing a new tool or view,
+which gives a different perspective on how to tackle the incident (knowledge
+sharing). This exercise can be repeated multiple times for different incidents.
+
+![Wheel of misfortune example](wheel.png)
+
+## Results
+
+The Observability Game Day has already been completed by multiple Skyscanner
+teams, where each team observability expert (ambassador) runs the session. The
+participants have given extremely positive feedback, where 90% of the responders
+say that after the Game Day, they feel more confident debugging production
+systems and would love to have further sessions.
+
+- Hugely valuable to run against real services and to compare and contrast
+  different debugging methods. I'm certain everyone, regardless of skill level,
+  will have got something out of the session - I know I did! Thank you for
+  taking the time to set this up and promoting it for us - Dominic Fraser
+  (Senior Software Engineer)
```

Comment:
```suggestion
  taking the time to set this up and promoting it for us - Dominic Fraser
  (Senior Software Engineer)
```

Are those people OK with being named here, and if so, we can link their github accounts

Reply:
Yes, they are more than fine :) One of them has and the other not, should we still link the one?